app/main.py

```diff
@@ -1 +1,49 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name: str = name
+        self.appetite: int = appetite
+        self.is_hungry: bool = is_hungry
+        Animal.alive.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
This is a minor issue, but according to the checklist, you don't need to use `else` statement if it's not necessary. In your case, you can omit the `else` statement and write the return statement at the end of the function. This makes the code a bit cleaner and easier to read.